src/components/Demo.js

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import Title from "@components/typography/Title";
+
+import {
+  demoContainerPSectionTop,
+  demoContainerSectionTop,
+  demoSectionTop,
+} from "@styles/demo.module.css";
+
+
+const Demo = ({
+  children,
+  title,
+  subtitle,
+  ...props
+}) => {
```

Comment:
Jak już usuniesz te zmienne, to wtedy możesz dodatkowo skrócić i pominąć `return ()` - zamiast tego wrzuć zwracaj od razu 
```
(
    <div className={demo}>
      <container className={demoContainer}>
        <Title styling={demoContainerHeader}>{title}</Title>
        <p className={demoContainerP}>{subtitle}</p>
      </container>
      {children}
    </div>
  );
```